packages/expo-sms/android/src/main/java/expo/modules/sms/SMSModule.java

```diff
@@ -120,4 +110,16 @@ public void onHostPause() {
   public void onHostDestroy() {
     // do nothing
   }
+
+  private String constructRecipients(List<String> addresses) {
+    if (addresses.size() > 0) {
+      final StringBuilder addressesBuilder = new StringBuilder(addresses.get(0));
+      for (String address: addresses) {
```

Comment:
```suggestion
      for (String address : addresses) {
```

I would suggest configuring Android Studio to format files on save. It has some downsides, like there are files in the project that aren't formatted and editing them will lead to unnecessary changes, but apart from that it's the easiest way to make sure files are formatted properly.